controllers/addon_controller.go

```diff
@@ -126,6 +126,10 @@ func (r *AddonReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 		return reconcile.Result{}, ignoreNotFound(err)
 	}
 
+	// Log the addon spec and the checksum
+	changedStatus, newChecksum := r.validateChecksum(instance)
```

Comment:
This is the first thing we already do in processAddon
https://github.com/keikoproj/addon-manager/blob/ef27eb55c2f360f9aa6d24d4ca60c0117bdb79bf/controllers/addon_controller.go#L239